internal/daemon/controller/handlers/credentiallibraries/credentiallibrary_service_test.go

```diff
@@ -609,7 +609,7 @@ func TestGet(t *testing.T) {
 	lib, err := vault.NewCredentialLibrary(store.GetPublicId(), "vault/path",
 		vault.WithCredentialType("username_password"),
 		vault.WithMappingOverride(
-			vault.NewUserPasswordOverride(
+			vault.NewUsernamePasswordOverride(
```

Comment:
Should the `name` fields with `user-password` be updated to `username-password`?